docs/configuration/git-providers/github-cloud.md

```diff
@@ -1,33 +1,28 @@
 # GitHub Cloud
 
-To integrate with GitHub we will use a GitHub OAuth Application.
+## `values.yaml` configuration
 
-To create the application on GitHub, visit **Settings / Developer settings / OAuth Apps / New OAuth App** and create an application pointing to your local Codacy deployment URL.
+Set your configuration values for your GitHub instance on the `values.yaml` file:
 
-![GitHub Application](images/github-application.png)
+| Field                          | Value                     |
+| ------------------------------ | ------------------------- |
+| `global.github.enabled`        | `true`                    |
+| `global.github.app.name`       | See below how to generate |
+| `global.github.app.id`         | See below how to generate |
+| `global.github.app.privateKey` | See below how to generate |
 
-You can fill all the fields with the suggested text above or use your own text except for the field `Authorization callback URL` where you must insert your local Codacy deployment URL.
+**Please note that you must go to `http://codacy.example.com/admin/integrations`, select the desired provider and `Test & Save` your configuration for it to be applied.**
 
-The URL should contain the endpoint/IP, the protocol (HTTP or HTTPS), and, if applicable, the port where it is running.
+## Create a GitHub App
 
-Correct:
+To integrate with GitHub we use a GitHub App.
 
--   <http://your.codacy.url.com>
--   <http://your.codacy.url.com:9000>
--   <http://53.43.42.12gi>
--   <http://53.43.42.12:9000>
+Follow the guide to [create the application on GitHub](create-github-app.md).
```

Comment:
This links to the github app creation docs for github enterprise. It should link to the Github cloud docs

Reply:
According to @mrfyda, the process to create the GitHub App on GitHub Cloud and Enterprise is largely the same. I will figure out a way of sharing this information better between the two Git providers on PR https://github.com/codacy/chart/pull/350.